readthedocs/settings/base.py

```diff
@@ -73,7 +73,7 @@ def SHOW_DEBUG_TOOLBAR(self):
     PRODUCTION_DOMAIN = 'readthedocs.org'
     PUBLIC_DOMAIN = None
     PUBLIC_DOMAIN_USES_HTTPS = False
-    USE_SUBDOMAIN = False
+    USE_SUBDOMAIN = True
```

Comment:
Note to self: This should not be checked in

Reply:
Why do you need this for testing? If you are using Docker, this is already set as `True`

https://github.com/readthedocs/readthedocs.org/blob/502b9b35f19197c452b4e73d70f9b78c129d4098/readthedocs/settings/docker_compose.py#L16